db/migrate/20211230214819_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :articles do |t|
+      t.string :question
+      t.boolean :for_partners
+      t.boolean :for_banks
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
I see that you create and drop the article table. Could you remove these migrations from this PR so that migration history stays accurate and clear?

Reply:
Sure, I'll delete these.

Reply:
I deleted them.